_storage/overview.md

```diff
@@ -0,0 +1,204 @@
+---
+layout: storage
+permalink: /:collection/:path.html
+---
+# A decentralized storage architecture
+
+The Blockstack Network stores application data using a storage service called
+Gaia. Transactional metadata is stored on the Blockstack blockchain and user
+application data is stored in Gaia hub. Storing data off of the blockchain
```

Comment:
"Stored in **a** Gaia hub" or perhaps even better "stored in a network of Gaia hubs"?

Reply:
So, there is only a single hub where user data is stored. In terms of implementing the service, the designer might choose to set up several and load balance among them.  That is a service implementation detail.  Stet.